cdap-ui/app/directives/my-pipeline-resource/my-pipeline-resource.html

```diff
@@ -0,0 +1,32 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<div class="resource-group">
+  <label class="control-label"> Virtual Cores </label>
+  <select class="form-control"
+    ng-model="virtualCores"
+    ng-options="core as core for core in cores"
+    ng-change="onVirtualCoresChange()"
+    >
+  </select>
+</div>
+<div class="resource-group">
+  <label class="control-label"> Memory </label>
+  <input class="form-control"
```

Comment:
we're not giving any warning if they input invalid input anymore? such as negative number?